### PR TITLE
Issue 13: Break up loop lesson to fix  'make check' errors

### PR DESCRIPTION
**Pull request by janetriley**
Address the following `make check` issues for loops: 

* Document contains heading not specified in the template: For Loops
* Document contains heading not specified in the template: Lists
* Document contains heading not specified in the template: Processing Multiple Files
* The topic page should not have sub-headings outside of special blocks. If a topic needs sub-headings, it should be broken into multiple topics.

The loop lesson is now three topics: loops, lists, and multiple files.    

+  Broke up the loop lesson into three topics: loops, lists, and multiple files
+ inserted them in the file naming scheme, and renumbered subsequent files
+ updated the links in index, references, and instructors to match.

### CODE REVIEW

File: 03-lists.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+> are much more efficient.
+
+There are many ways to change the contents of in lists besides assigning to elements:
+
+~~~ {.python}
+odds.append(11)
+print 'odds after adding a value:', odds
+~~~
+
+FIXME: output
```
Comment (kynan) [inline 03-lists.md]: Is this the opportunity to address those FIXMEs?

File: reference.html [hunk 2]
```diff
@@ -118,184 +124,184 @@ <h2 id="errors-and-exceptions"><a href="07-errors.html">Errors and Exceptions</a
 <h2 id="glossary">Glossary</h2>
 <dl>
 <dt><span id="additive-color-model">additive color model</span></dt>
-<dd>A way to represent colors as the sum of contributions from primary colors such as <a href="#rgb">red, green, and blue</a>.
+<dd><p>A way to represent colors as the sum of contributions from primary colors such as <a href="#rgb">red, green, and blue</a>.</p>
```
Comment (kynan) [inline reference.html]: Are those due to use of a different pandoc version for the translation?

File: 03-lists.md (hunk 1 above)
Comment (janetriley) [inline 03-lists.md]: I'd prefer to submit the file changes first, and take on the other tickets to break up topics.  Renaming files affects everyone's work, and small changes are easier to approve.

But yes, the FIXMEs bug me. I like clean test runs.

**Comment by kynan** [comment]
These changes make sense to me, only minor niggles.

File: reference.html (hunk 2 above)
Comment (janetriley) [inline reference.html]: I don't know. I have pandocfilters version 1.2.3 installed.  Requirements.txt doesn't specify a version.

**Comment by janetriley** [comment]
Thank you. I'll merge it in and move to the next issue.

File: reference.html (hunk 2 above)
Comment (r-gaia-cs) [inline reference.html]: > Are those due to use of a different pandoc version for the translation?

Probably yes.

> I have pandocfilters version 1.2.3 installed.

This is a very old version of Pandoc. The last version is 1.13.2. You can download the last version at https://github.com/jgm/pandoc/releases.

**Pull request merged**